backend/backend/services/openAI/prompts/uniswap_governance.py

```diff
@@ -0,0 +1,69 @@
+from backend.services.openAI.graph_prompt import GraphPromptBase as Prompt
+
+
+EXAMPLES = [
+
+Prompt(
+q="Who are the top delegates by voting power?",
+o="""query{
+  votes(orderBy: voter__tokenHoldersRepresentedAmount, orderDirection: desc) {
+    voter {
+      tokenHoldersRepresentedAmount
+      id
+    }}}
+    """),
+Prompt(
+q="What is the total number of proposals, number of proposals queued for execution, number of proposals executed, and number of proposals canceled for a governance with the ID 11?",
```

Comment:
i think you mean number of votes for prop with id 11...?

Reply:
do we need `uniswap_governance.py`? they all seem off or a copy of `compound_governance.py`